tests/integration/test_hitl_loop_resume_fix.py

```
@@ -0,0 +1,649 @@
+            # Initialize state
+            Step(
+                name="init_state",
+                agent={"id": "flujo.builtins.passthrough"},
+                input="",
+                updates_context=True,
+                sink_to="scratchpad",
+                processors={
+                    "output_processors": [
+                        {
+                            "type": "callable",
+                            "callable": lambda x: {"slots": {}, "initial_goal": ""},
```

Ruff (0.13.3)

Unused lambda argument: `x`

(ARG005)